blockchain/tx_list_test.go

```diff
@@ -195,11 +195,11 @@ func TestSubstituteTransactionAbort(t *testing.T) {
 	oldTx := pricedTransaction(0, 21000, big.NewInt(50), key)
 	newTx := pricedTransaction(0, 21000, big.NewInt(40), key)
 
-	if result, _ := txList.Add(oldTx, DefaultTxPoolConfig.PriceBump); !result {
+	if result, _ := txList.Add(oldTx, DefaultTxPoolConfig.PriceBump, true); !result {
 		t.Error("it cannot add tx in tx list.")
 	}
 
-	if result, replaced := txList.Add(newTx, DefaultTxPoolConfig.PriceBump); result || replaced != nil {
+	if result, replaced := txList.Add(newTx, DefaultTxPoolConfig.PriceBump, true); result || replaced != nil {
```

Comment:
Could you add test code for cancel transaction (Whatever the gas price is, if it is greater than the base rate, replace it with the old tx)?
Or is there any test code for that?

Reply:
Sure, I'll try it

Reply:
The testcase is added in `TestCancelTransactionAccepted`